bin/sandbox

```diff
@@ -33,13 +33,28 @@ fi
 
 extension_name="solidus_starter_frontend"
 
+if [[ -z $GENERATE_FRONTEND ]]; then
+  sandbox_name='sandbox'
+  extension_gem_line="gem '$extension_name', path: '..'"
+else
+  sandbox_name='sandbox-generated'
+  extension_gem_line=""
+
+  if ! gem list --local | grep $extension_name; then
+    echo "ERROR: Please install $extension_name to generate the frontend of the sandbox."
+    exit 1
+  fi
```

Comment:
`solidus_starter_frontend` must be installed globally in order to create the sandbox-generated app.

Reply:
The problem I see is that the developer probably doesn't want to install the released version of `solidus_starter_frontend`. Instead, they want to check that their changes didn't break the installation process. They can indeed install the extension by `gem build + gem install /local/gem/file`, but I think we can automate it. See my comment below.